src/shared/lib/sortedFindWith.spec.ts

```diff
@@ -11,22 +11,34 @@ describe('sortedFindWith', () => {
     it('one element array', () => {
         const array = [{ id: 0, name: 'hey' }];
         assert.isUndefined(sortedFindWith(array, t => Math.sign(t.id + 1)));
-        assert.deepEqual(sortedFindWith(array, t => Math.sign(t.id)), {
-            id: 0,
-            name: 'hey',
-        });
+        assert.deepEqual(
+            sortedFindWith(array, t => Math.sign(t.id)),
+            {
+                id: 0,
+                name: 'hey',
+            }
+        );
     });
     it('two element array', () => {
-        const array = [{ id: 0, name: 'hey' }, { id: 1, name: 'yo' }];
```

Comment:
https://prettier.io/blog/2019/11/09/1.19.0.html#break-arrays-of-arraysobjects-if-each-element-has-more-than-one-elementproperty-6694-by-sosukesuzuki